File: Johanna Schreiner/wk01/3-wed/arrays.js


var daysOfTheWeek = ["Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"];
console.log(daysOfTheWeek);


daysOfTheWeek.pop();
console.log(daysOfTheWeek);


daysOfTheWeek.unshift("Sunday");
console.log(daysOfTheWeek);


var newDaysOfTheWeek = [["Monday","Tuesday","Wednesday","Thursday","Friday"],["Saturday","Sunday"]];
console.log(daysOfTheWeek);


newDaysOfTheWeek.pop();
console.log(daysOfTheWeek);


newDaysOfTheWeek[0].sort();
console.log(daysOfTheWeek);

File: Johanna Schreiner/wk04/1-mon/arrays/arrays.rb
require 'pry'

# ARRAYS ONE

days_of_the_week = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]

sunday_first = days_of_the_week.pop

days_of_the_week.unshift(sunday_first)

# puts(days_of_the_week)


# ARRAYS TWO

days_of_the_week_2 = [["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"], ["Saturday", "Sunday"]]

# ARRAYS THREE
days_of_the_week_2.shift

weekends = days_of_the_week_2

# puts(weekends)

# ARRAYS FOUR

puts(weekends.sort)
